### DISCUSSION TITLE
Saturation Curves from Robyn Response

### DISCUSSION THREAD

**Discussion by tgtod002** [facebookexperimental/Robyn#795] [Q&A]
can someone explain the plot produced by Robyn Response.   More particularly what is the difference between the Asterisks and the dots in the plot.

![image](https://github.com/facebookexperimental/Robyn/assets/69650410/20ecea4f-f687-4873-ac00-f215f646569b)

**Comment by gufengzhou** [comment]
The asterisks are the "carryover points" and the dots are the actual spends on top.
